doc/RELEASE_NOTES.md

```diff
@@ -1,11 +1,13 @@
 This is a very *significant* "minor" release of the pbench-agent code base, primarily to deliver the new "Tool Meister" sub-system.
 
-_*NOTE WELL*_: The notion of a "default" tool set is being deprecated and will be removed in the upcoming Pbench Agent v1.0 release, along with the addition of a few named tool sets.  See below.
+_*NOTE WELL*_:
+
+ * The notion of a "default" tool set is being deprecated and will be removed in the upcoming Pbench Agent v1.0 release, along with the addition of a few named tool sets.  See "Default Tool Set is _*Deprecated*_; Named tool sets introduced" below.
```

Comment:
This sentence somewhat implies *removal* of "a few named tool sets". Maybe break up the sentence? "The notion of a 'default' tool set [...] will be removed ... To replace it, Pbench is introducing a set of named tool sets." ???

Reply:
Good point.